tests/losses/test_unified_focal_loss.py

```
@@ -12,55 +12,98 @@
+        TEST_CASE_BINARY_LOGITS,
+        TEST_CASE_BINARY_2CH,
+        TEST_CASE_MULTICLASS_PERFECT
+    ])
+    def test_perfect_prediction(self, input_data, expected_val, args):
+        loss_func = AsymmetricUnifiedFocalLoss(**args)
+        result = loss_func(**input_data)
+        # We use a small tolerance because 10.0 logits is not exactly probability 1.0
+        np.testing.assert_allclose(result.detach().cpu().numpy(), expected_val, atol=1e-3, rtol=1e-3)
+
+    @parameterized.expand([TEST_CASE_MULTICLASS_WRONG])
+    def test_wrong_prediction(self, input_data, expected_val, args):
```

Ruff (0.14.10)

Unused method argument: `expected_val`

(ARG002)